### PR TITLE
adding proposal for cep24

### PR DESCRIPTION
**Pull request by gidden**
hi folks, another cep for you all related to preference adjustment

### CODE REVIEW

File: source/cep/cep24.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+:Author: Matthew Gidden
+:Status: Proposed
+:Type: Process Track
+:Created: 2015-04-09
+
+
+Abstract
+========
+
+This CEP is meant to be somewhat of a work in progress in the near term. The immediate concerns involved default preferences and preference adjustment with regard to implementing international trade instruments.
```
Comment (scopatz) [inline source/cep/cep24.rst]: I am not comfortable with "work in progress" CEPs. I think CEPs should be concrete proposals that may be modified in response to the review process.  But "<fill in later>" CEPs goes somewhat against the purpose of having proposals.

**Comment by gidden** [comment]
addressed

**Comment by scopatz** [comment]
Are you going to post these to the list?

File: source/cep/cep24.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+=========
+
+The default preference will change from zero to unity. Default preferences will
+then be stable under both additive and multiplicative operations. Negative
+preferences will still denote potential trades to be removed from resource
+exchanges before the solution phase.
+
+Further, in order for managers of bidding regions to apply tariffs, or any other
+inter-entity instruments, they must be involved in preference
+adjustment. Previously, only managers of the bidding entities were allowed to
```
Comment (rwcarlsen) [inline source/cep/cep24.rst]: s/bidding/requesting/

File: source/cep/cep24.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+affect preference values. This CEP will allow managers of requesting entities to
```
Comment (rwcarlsen) [inline source/cep/cep24.rst]: s/requesting/bidding/

**Comment by rwcarlsen** [comment]
Just to be sure, does the managers of bidding entities include the bidders (i.e. does the actual bidder get to adjust preferences)?  If the answer is yes, then one possible use case would be for facilities with impending decommissioning to increase their preference in order to liquidate material stocks before dying.

**Comment by gidden** [comment]
This proposal does not include the bidding entities. It is meant to enable
institutional and regional effects of bidding entities.

On Mon, Apr 27, 2015 at 12:31 PM, Robert Carlsen <notifications@github.com>
wrote:

> Just to be sure, does the managers of bidding entities include the bidders
> (i.e. does the actual bidder get to adjust preferences)? If the answer is
> yes, then one possible use case would be for facilities with impending
> decommissioning to increase their preference in order to liquidate material
> stocks before dying.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus.github.com/pull/130#issuecomment-96751311>
> .
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192

**Comment by rwcarlsen** [comment]
Is there any particular reason for the asymmetry?  Requesters get to adjust prefs before their parents.  I would prefer to have the bidder included in the bid-side chain of pref adjustments.

**Comment by gidden** [comment]
The proposed implementation is required to model motivating use cases as
stated by our team (specifically, region and institution-affected flows).
The model as currently described is "consumer-centric". Adding bidders to
the chain of preference adjusters would change that underlying model.

I think that there could be a possible future that includes a
"supplier-centric" exchange feature as well as a "competition"-like
exchange that include input from both suppliers and consumers. But that is
not the point of this CEP, and should be addressed in a future CEP.

On Mon, Apr 27, 2015 at 12:42 PM, Robert Carlsen <notifications@github.com>
wrote:

> Is there any particular reason for the asymmetry? Requesters get to adjust
> prefs before their parents. I would prefer to have the bidder included in
> the bid-side chain of pref adjustments.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus.github.com/pull/130#issuecomment-96753504>
> .
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192

**Comment by rwcarlsen** [comment]
Seems reasonable.  Thanks for the discussion.  I like this CEP and think it is a good step forward. +1

**Pull request merged**